### ISSUE TITLE
A user should not get tasks for validation, which were done by him-/herself

### ISSUE THREAD

**Issue by torbjoernk** [hotosm/osm-tasking-manager#49]
Given a user has chosen a task, worked on it and submitted it as done. This task is now marked as done and is awaiting validation. Given further, that this user has completed (e.g "done") a couple of tasks. And now, this user decides to validate some other tasks. This user should **not** get one of the tasks for validation, which were marked as done by this user.

_I'm opening an issue for this, as I encountered such a situation just yesterday with HOT's task manager._

**Comment by bigfatfrog67** [comment]
Yes, strongly agree.  It's a real pain to be constantly presented with your own tasks to validate.
